monitoring/update-bucket-capacity-info-cronjob/alerts.test.yaml

```diff
@@ -0,0 +1,53 @@
+evaluation_interval: 1m
+rule_files:
+  - alerts.rendered.yaml
+
+tests:
+  - name: Update Bucket Capacity Info CronJob Test Taking Too Long
+    interval: 1m
+    input_series:
+      - series: 'kube_job_status_start_time{job="kube-state-metrics", job_name="artesca-data-ops-update-bucket-capacity-info-1", namespace="zenko"}'
+        values: '0x9'
+      - series: 'kube_job_status_completion_time{job="kube-state-metrics", job_name="artesca-data-ops-update-bucket-capacity-info-1", namespace="zenko"}'
+        values: '_x4 240x5'
+      - series: 'kube_job_status_start_time{job="kube-state-metrics", job_name="artesca-data-ops-update-bucket-capacity-info-2", namespace="zenko"}'
+        values: '_x4 240x5'
+      - series: 'kube_job_status_completion_time{job="kube-state-metrics", job_name="artesca-data-ops-update-bucket-capacity-info-2", namespace="zenko"}'
+        values: '_x9 '
+    alert_rule_test:
+      - alertname: UpdateBucketCapacityJobTakingTooLong
+        eval_time: 4m
+        exp_alerts: []
+      - alertname: UpdateBucketCapacityJobTakingTooLong
+        eval_time: 9m
+        exp_alerts:
+          - exp_labels:
+              severity: warning
+              job_name: artesca-data-ops-update-bucket-capacity-info-1
+            exp_annotations:
+              description: |
+                Job artesca-data-ops-update-bucket-capacity-info is taking more than 4m to complete.
```

Comment:
Same comment here, considering that the doc example it changing to 5min (and many readers may blindly run this command)